### PR TITLE
Updated handle click in button form

### PR DESCRIPTION
**Pull request by annamarika**
I added an edit function to my products.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -22,6 +21,7 @@ export default function Home({ initialProduct }) {
 	});
 
 	const [products, setProducts] = useState(loadFromLocal('localProducts') ?? initialProduct);
+	console.log(products);
```
Comment (HishamAlmannai) [inline pages/index.js:24]: Please remove if not necessary

File: src/components/Product/Product.jsx [hunk 2]
```diff
@@ -1,33 +1,169 @@
...
+		formState: { errors },
+	} = useForm();
+
+	const {
+		image: imageValue,
+		detail: detailValue,
+		email: emailValue,
+		title: titleValue,
+	} = watch();
+
```
Comment (HishamAlmannai) [inline src/components/Product/Product.jsx:83]: Is this `watch()` necessary?

File: src/components/Product/Product.jsx [hunk 3]
```diff
@@ -1,33 +1,169 @@
...
+							})}
+						/>
+					</InputSingleContainer>
+				</InputContainer>
+				<ButtonContainer>
+					<DefaultButton
+						type="submit"
+						onClick={event => {
+							event.preventDefault();
+							onUpdateProduct(
```
Comment (HishamAlmannai) [inline src/components/Product/Product.jsx:152]: Just curious: what does the "on.." in your wording stand for?

**Review by HishamAlmannai — APPROVED** [review]
Good job! Just some minor changes.

File: src/components/Product/Product.jsx (hunk 2 above)
Comment (annamarika) [reply src/components/Product/Product.jsx:83]: yes it's necessary, on line 153 you can see that I'm using the *Values to overwrite my props. And you need watch when using form hook to "watch your values". If you're not using form hook you can work with usestate and create a usestate for every single prop value

File: src/components/Product/Product.jsx (hunk 3 above)
Comment (annamarika) [reply src/components/Product/Product.jsx:152]: this is how we learned it. This is about state uplifting. In the index.js I'm creating an updateProduct function and I'm adding an onUpdateProduct={updateProduct} attribute to my productGrid

**Review by cintiavazquez — APPROVED** [review]

**Comment by cintiavazquez** [comment]
You don't need different storybook files for the button, you can display several variants in one story :) Other than that, great work!!

**Pull request merged**